File: App/src/app/_core/_model/result-of-month.ts
export interface ResultOfMonth {
  id: number;
  title: string;
  month: number;
  createdBy: number;
  modifiedBy: number | null;
  createdTime: string;
  modifiedTime: string | null;
}
export interface ResultOfMonthRequest {
  objectiveId: number;
  id: number;
  title: string;
  createdBy: number;

}

File: A4KPI/Models/ResultOfMonth.cs
using A4KPI.Models.Interface;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace A4KPI.Models
{
    [Table("ResultOfMonth")]
    public class ResultOfMonth: IDateTracking
    {
        [Key]
        public int Id { get; set; }
        public string Title { get; set; }
        public int Month { get; set; }
        public int ObjectiveId { get; set; }
        public int CreatedBy { get; set; }
        public int? ModifiedBy { get; set; }
        public DateTime CreatedTime { get; set; }
        public DateTime? ModifiedTime { get; set; }
        [ForeignKey(nameof(CreatedBy))]
        public virtual Account Account { get; set; }
        [ForeignKey(nameof(ObjectiveId))]
        public virtual Objective Objective { get; set; }
    }
}
